diagram/__init__.py

```diff
@@ -1,5 +1,6 @@
 ﻿from __future__ import absolute_import
 from .plantuml import PlantUMLProcessor
+from .sublime3.py import Sublime3Viewer
```

Comment:
On my Windows installation, I had to change `.sublime3.py` to `.sublime3` to avoid the following error when loading the plugin:
`ImportError: No module named 'sublime_diagram_plugin.diagram.sublime3.py'; sublime_diagram_plugin.diagram.sublime3 is not a package`